File: KARATENOVO_SELENIUM/src/test/java/com/selenium/tests/ExampleTest.java
package com.selenium.tests;

import com.selenium.framework.BaseTest;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class ExampleTest extends BaseTest {

    @Test
    public void verifyGoogleTitle() {
        driver.get("https://www.google.com");
        String title = driver.getTitle();
        System.out.println("Page title is: " + title);
        Assertions.assertTrue(title.contains("Google"), "Title should contain 'Google'");
    }
}

File: KARATENOVO_SELENIUM_CSHARP/ExampleTest.cs
using NUnit.Framework;
using OpenQA.Selenium;

namespace SeleniumTests
{
    public class ExampleTest : BaseTest
    {
        [Test]
        public void VerifyGoogleTitle()
        {
            driver.Navigate().GoToUrl("https://www.google.com");
            string title = driver.Title;
            System.Console.WriteLine("Page title is: " + title);
            Assert.That(title, Does.Contain("Google"), "Title should contain 'Google'");
        }
    }
}
